__fixtures__/with-git-protocol-dep/pnpm-lock.yaml

```diff
@@ -0,0 +1,26 @@
+lockfileVersion: '6.0'
+
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
+
+dependencies:
+  is-negative:
+    specifier: github:kevva/is-negative#master
+    version: github.com/kevva/is-negative/219c424611ff4a2af15f7deeff4f93c62558c43d
+
+packages:
+
+  /is-positive@3.1.0:
+    resolution: {integrity: sha512-8ND1j3y9/HP94TOvGzr69/FgbkX2ruOldhLEsTWwcJVfo4oRjwemJmJxt7RJkKYH8tz7vYBP9JcKQY8CLuJ90Q==}
+    engines: {node: '>=0.10.0'}
+    dev: false
+
+  github.com/kevva/is-negative/219c424611ff4a2af15f7deeff4f93c62558c43d:
+    resolution: {registry: http://localhost:4873/, tarball: https://codeload.github.com/kevva/is-negative/tar.gz/219c424611ff4a2af15f7deeff4f93c62558c43d}
```

Comment:
Why is there a registry field here? There should be no registry field for the git-hosted resolutions.

But even if I am not right, then I guess the registry field would have to be `https://github.com`

Reply:
Thanks for pointing out the problem here. It should be my mistake here. I can't remember where it came from.